joblib/pool.py

```diff
@@ -6,7 +6,7 @@
 
 This module should not be imported if multiprocessing is not
 available as it implements subclasses of multiprocessing Pool
-that uses a custom alternative to SimpleQueue.
+which uses a custom alternative to SimpleQueue.
```

Comment:
I don't think we should start splitting hair about that vs which. You can google it though and potentially learn something (I just did).